### DISCUSSION TITLE
Question about dagster-postgres version mapping

### DISCUSSION THREAD

**Discussion by dagsir** [dagster-io/dagster#17768] [Q&A]
Hi all,

I'm using dagster `"1.1.20"` and dagster-postgres `"0.17.20"` . What is the latest version of `bitnami/postgresql:` I can connect to? Where can I find the mapping between the `dagster-postgres` versions and the `postgres` versions? Looked in source code, but unlucky so far. Thanks a lot in advance :slightly_smiling_face:

Jori

---

*The question was originally asked in [Dagster Slack](https://dagster.slack.com/archives/C01U954MEER/p1699358188621539?thread_ts=1699358188.621539&cid=C01U954MEER).*

**Comment by matboett** [comment]
hi there,
we are using dagster 1.5.9 and dagster postgres 0.21.9 and in the helm values.yaml included there's still a 14.6 referenced (10 months old on this day)

are you trying to get an answer on this "outdated" issue as well, the newest Postgres version 14.x is 14.10 (released just 3 days ago)

according to the values.yaml of v1.1.20 the bitnami Postgres image tag was 14.6 back then as well

https://github.com/dagster-io/dagster/blob/8c9f54a2b4c63aa156d1aa328deecf8df90dac27/helm/dagster/values.yaml#L746
